pkg/k8shandler/common.go

```diff
@@ -242,16 +242,18 @@ func newProxyContainer(imageName, clusterName string) (v1.Container, error) {
 			},
 		},
 		Args: []string{
+			// HTTPS default listener for Elasticsearch
 			"--listening-address=:60000",
```

Comment:
shouldn't this be `9200`? the logging service is configured to point to port `9200` for http ES traffic.

Reply:
This is the port we expose to other services not the Elasticsearch port. For the Elasticsearch port you need to take a look on the `ElasticsearchURL` config options of elasticsearch-proxy

Reply:
@ewolinetz the services listens on a given port which is 9200 but points to the proxy which will listen on 60000.  The proxy will then point to ES which is 9200 but only accepts traffic from the proxy